google-cloud-vision/docs/toc.json

```diff
@@ -32,24 +32,12 @@
     {
       "title": "Vision",
       "type": "google/cloud/vision",
-      "patterns": ["\/cloud\/vision", "google\/protobuf", "google\/rpc", "google\/type"],
```

Comment:
In `Rakefile`, the TOC configuration for the Angular docs app includes:

```ruby
  title: "Google::Rpc",
              include: ["google/rpc"]
```

If the `"google\/rpc"` pattern is removed here and below from `patterns`, I believe the left nav active state will be broken when docs under that path are selected.

Reply:
I'm not familiar with how this file works. What does `patterns` do?

I've added these back, following the `video_intelligence` `patterns` as a template, since I assume this should look analogous.

Reply:
If a regex in `patterns` is a match for the URL in the Angular app, the left nav will be active (open) for the resource (page). The behavior of the left nav was previously very simplistic and did not support paths outside of the library root (e.g., `google/cloud/vision`), so to support alternate paths(e.g., `google/protobuf`), we added this mechanism.